File: socialmedia/src/main/java/com/mirceanealcos/socialmedia/dto/post/PostDto.java
package com.mirceanealcos.socialmedia.dto.post;

import com.mirceanealcos.socialmedia.dto.comment.CommentDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PostDto {
    private Long id;
    private String title;
    private String content;
    private String status;
    private String creationDate;
    private String userName;
    private Long userId;
    private List<CommentDto> comments;
}

File: socialmedia-desktop-client/objects/PostDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace socialmedia_desktop_client.objects
{
    internal class PostDto
    {

        public long id {  get; set; }
        public string title { get; set; }
        public string content { get; set; }
        public string status {  get; set; }
        public string userName {  get; set; }
        public long userId { get; set; }


        public PostDto() { }

        public PostDto(long id, string title, string content, string status, string userName, long userId)
        {
            this.id = id;
            this.title = title;
            this.content = content;
            this.status = status;
            this.userName = userName;
            this.userId = userId;
        }
    }
}
